examples/Calculating Strain via Voxelization.ipynb

```diff
@@ -119,14 +119,14 @@
     "for i in range(len(framefiles)):\n",
```

Comment:
<p>There's no NearestNeighbors anymore. @mikemhenry offered to set up nbval for us, it might be a good time to add that testing to make sure we haven't missed other API changes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>